reports/test_emails.csv

```diff
@@ -0,0 +1,5 @@
+﻿test_email.csv,,,,

```

Comment:
This was based off of the google doc included in the original script that contains a title row. I wanted the test file to mimic the google doc as much as possible

Reply:
This is necessary to mock the production sheet. Let me double check and document our discussion on splitting the creation of a csv vs using the script to pull the underlying data (since the cleaning has to happen somewhere). I think all the asynchronous feedback on the PR and slack without a clear end-to-end spec risks landing the wrong thing / wasting your time on PR cycles.

Reply:
The production sheet has a header row

![image](https://user-images.githubusercontent.com/2574498/146800016-1f886fec-fdb0-44cd-8de2-3d362659b6b8.png)

Can we merge this as is, so we can support the sending of this sheet, and open an issue for refining the process? It's unclear what the goal / process is, if it isn't to send emails based on the production sheet.

I could see once this is off the ground...

* creating a separate script to pull and pre-process the production sheet (whether in data-analyses or otherwise)
* defining an input CSV format (e.g. whatever the above script will produce)

See this discussion for context:

![image](https://user-images.githubusercontent.com/2574498/146800438-db31a7b3-9251-4278-aaee-de0711a6eca1.png)

Reply:
I didn't fully connect all the dots, so since the production sheet is like this, we'll keep it as is.